homeassistant/components/deconz/strings.json

```diff
@@ -23,15 +23,15 @@
       }
     },
     "error": {
-      "no_key": "Couldn't get an API key"
+      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]"
     },
     "abort": {
       "already_configured": "Bridge is already configured",
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "no_bridges": "No deCONZ bridges discovered",
       "no_hardware_available": "No radio hardware connected to deCONZ",
       "not_deconz_bridge": "Not a deCONZ bridge",
-      "one_instance_only": "Component only supports one deCONZ instance",
```

Comment:
This one can be removed since it is no longer used

Reply:
ok

Reply:
@Kane610 any idea why the test is failing on `tests/components/light`? not related to the PR
https://github.com/home-assistant/core/pull/41978/checks?check_run_id=1268246233
```
Check failure on line 100 in tests/components/light/test_device_trigger.py

@github-actions
github-actions
/ Run tests Python 3.8 (group 2)

tests/components/light/test_device_trigger.py#L100
test_if_fires_on_state_change[pyloop]

assert False
```